share/templates.d/99-generic/runtime-postinstall.tmpl

```diff
@@ -117,8 +117,3 @@ remove etc/lvm/lvm.conf
 append etc/lvm/lvm.conf "global {\n\tuse_lvmetad = 1\n}\n"
 
 ## TODO: we could run prelink here if we wanted?
-
```

Comment:
because this drops xorg-x11-fonts-misc, where `/usr/share/X11/fonts` came from. but now I look more closely, I guess we should just drop `/usr/share/X11/fonts` from the command but leave the rest of it?